README.rst

```diff
@@ -182,6 +182,11 @@ Authors
 Please check the `contribution statistics for a list of
 contributors <https://github.com/scikit-hep/pyhf/graphs/contributors>`__.
 
+Milestones
+----------
+
+- 2020-07-28: 1000 GitHub issues and pull requests. (See #1000)
```

Comment:
```suggestion
- 2020-07-28: 1000 GitHub issues and pull requests. (See PR :pr: `#1000`)
```

I think we need this given

```
git grep "\:pr\:"

```

Reply:
@lukasheinrich  Oops this is still wrong. Sorry! Let me debug a bit more.

Reply:
@kratsg @lukasheinrich look better now that I fixed my error? :)

![1000_events](https://user-images.githubusercontent.com/5142394/88603806-90d8da00-d03b-11ea-9bab-6f69105a31c2.png)